lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -207,96 +198,112 @@ shutdown_vlan:
     default_value: true
 
 speed:
-  config_get: "show running interface all"
-  config_get_token: ['/^interface %s$/i', '/^speed (.*)$/']
-  config_set: ["interface %s", "speed %s"]
+  _exclude: [cli_ios_xr]
+  config_get_token_append: ['/^speed (.*)$/']
+  config_set_append: ["speed <speed>"]
   default_value: "auto"
 
 svi_autostate:
+  _exclude: [cli_ios_xr]
   kind: boolean
   config_get_token_append: '/^(?:no )?autostate$/'
-  config_set_append: "%s autostate"
+  config_set_append: "<state> autostate"
   default_value: true
-  test_config_result:
-    false: false
-    true: true
 
 svi_management:
+  _exclude: [cli_ios_xr]
   kind: boolean
   config_get_token_append: '/^management$/'
-  config_set_append: "%s management"
+  config_set_append: "<state> management"
   default_value: false
 
 switchport:
+  _exclude: [cli_ios_xr]
   kind: boolean
   config_get_token_append: '/^switchport$/'
-  config_set_append: "%s switchport"
+  config_set_append: "<state> switchport"
   # default_value: n/a. This is derived from system_default_switchport
 
 switchport_autostate_exclude:
+  _exclude: [cli_ios_xr]
   kind: boolean
   config_get_token_append: '/(^switchport autostate exclude)/'
-  config_set_append: "%s switchport autostate exclude"
+  config_set_append: "<state> switchport autostate exclude"
   default_value: false
 
 switchport_mode_dot1q_tunnel:
-  test_config_result:
-    false: RuntimeError
+  default_only: ""
```

Comment:
As currently implemented for Nexus, interfaces that don't support switchport mode report as `""` (which gets mapped to `:disabled` in the class) instead of `nil`. I'm just following that pattern for all interfaces on XR. Based on our conversations over the last few days, changing this to report `nil` for unsupported interfaces is probably desirable but since that will have impact to Nexus as well, I'd like to do that in a subsequent commit. Sound reasonable?